src/DIRAC/Core/Utilities/Subprocess.py

```diff
@@ -153,17 +153,17 @@ class Subprocess(object):
 
     """
 
-    def __init__(self, timeout=False, bufferLimit=52428800):
+    def __init__(self, timeout=False, bufferLimit=20971520):
         """c'tor
 
         :param int timeout: timeout in seconds
-        :param int bufferLimit: buffer size, default 5MB
+        :param int bufferLimit: buffer size, default 20MB
         """
         self.log = gLogger.getSubLogger("Subprocess")
         self.timeout = False
         try:
             self.changeTimeout(timeout)
-            self.bufferLimit = int(bufferLimit)  # 5MB limit for data
+            self.bufferLimit = int(bufferLimit)  # 20MB limit for data
```

Comment:
remove this comment here, so there is just one place that needs updating
```suggestion
            self.bufferLimit = int(bufferLimit)
```